src/azure-cli/azure/cli/command_modules/acs/agentpool_decorator.py

```diff
@@ -1504,6 +1504,19 @@ def get_disable_windows_outbound_nat(self) -> bool:
         """
         return self._get_disable_windows_outbound_nat()
 
+    def get_ignore_pod_disruption_budget(self) -> bool:
+        return self._get_ignore_pod_disruption_budget()
+
+    def _get_ignore_pod_disruption_budget(self) -> bool:
+        """Obtain the value of ignore_pod_disruption_budget, default value is False.
+
+        :return: bool
+        """
+        # read the original value passed by the command
+        ignore_pod_disruption_budget = self.raw_param.get("ignore_pod_disruption_budget", False)
+
+        # This parameter does not need dynamic completion.
+        return ignore_pod_disruption_budget
```

Comment:
I think you don't need this as this is not an option for `az aks nodepool add/update`

Reply:
I see. I overlooked your comment in the code. Removed.